core/src/main/scala/org/apache/spark/sql/execution/columnar/CachedBatchHolder.scala

```diff
@@ -16,11 +16,19 @@
  */
 package org.apache.spark.sql.execution.columnar
 
+import java.sql.PreparedStatement
+
+import scala.collection.mutable.ArrayBuffer
+
 import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.execution.columnar.impl.{ColumnFormatRelation, IndexColumnFormatRelation}
```

Comment:
it already showing up with Index*Rel in toString. Have fixed that in BaseColumnFormatRelation.toString